### PR TITLE
[risk=low][no ticket] A few small updates to #6751

### PR DESCRIPTION
**Pull request by jmthibault79**
I make a few small tweaks related to RDR backfill/testing while I was working on the Demo Survey V2 RDR export.  This PR has those tweaks independent of Demo Survey work.

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have added explanatory comments where the logic is not obvious
- [ ] I have run and tested this change locally, and my testing process is described here
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later
- [ ] If this includes an API change, I have run the E2E tests on this change against my local server with [yarn test-local](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples) because this PR won't be covered by the CircleCI tests 
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P)
- [ ] If this change impacts deployment safety (e.g. removing/altering APIs which are in use) I have documented these in the description
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and updated the appropriate API consumers
  * AoU UI
  * [Perf tests](https://github.com/broadinstitute/mcnulty/blob/develop/src/test/scala/services/AoU.scala)
  * [Researcher Directory export](https://github.com/all-of-us/workbench/wiki/Researcher-Directory-(RDR-export))
  * Cron tasks - for Offline*Controllers
  * SumoLogic - for EgressAlert

### CODE REVIEW

File: api/libproject/devstart.rb [hunk 1]
```diff
@@ -2016,7 +2016,7 @@ def invalidate_rdr_export(cmd_name, *args)
   end
 
   get_user_confirmation(
-    "RDR export invalidation should rarely be used; for backfilling workspace data you " +
```
Comment (jmthibault79) [inline api/libproject/devstart.rb:2019]: @calbach could you confirm that this change is correct?

File: api/src/test/java/org/pmiops/workbench/rdr/RdrExportServiceImplTest.java [hunk 2]
```diff
@@ -192,33 +191,36 @@ private DbWorkspace buildDbWorkspace(
...
     doNothing().when(mockRdrApi).exportResearchers(anyList(), anyBoolean());
 
-    rdrExportService.exportUsers(
-        ImmutableList.of(dbUserWithEmail.getUserId(), dbUserWithoutEmail.getUserId()), false);
+    List<Long> userIds =
+        ImmutableList.of(dbUserWithEmail.getUserId(), dbUserWithoutEmail.getUserId());
+    rdrExportService.exportUsers(userIds, backfill);
 
-    verify(rdrExportService, times(1)).updateDbRdrExport(any(), anyList());
+    verify(rdrExportService, times(1)).updateDbRdrExport(RdrEntity.USER, userIds);
```
Comment (jmthibault79) [inline api/src/test/java/org/pmiops/workbench/rdr/RdrExportServiceImplTest.java:201]: we can be more specific here

File: api/libproject/devstart.rb (hunk 1 above)
Comment (calbach) [reply api/libproject/devstart.rb:2019]: Thanks, yes - left in accidentally.

**Review by calbach — APPROVED** [review]
Thanks

**Pull request merged**